src/components/ValuePicker/ValueSelectorModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {useMemo} from 'react';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Modal from '@components/Modal';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import type {ValueSelectorModalProps} from './types';
+
+function ValueSelectorModal({items = [], selectedItem, label = '', isVisible, onClose, onItemSelected, shouldShowTooltips = true}: ValueSelectorModalProps) {
+    const styles = useThemeStyles();
+
+    const sectionsData = useMemo(() => [{data: items.map((item) => ({...item, isSelected: item === selectedItem, keyForList: item.value ?? ''}))}], [items, selectedItem]);
```

Comment:
Please keep original keys. Not sure why you changed them
`{value: item.value, alternateText: item.description, keyForList: item.value, text: item.label, isSelected: item === selectedItem}`

And keep original variable name `sections` unless any reason

Reply:
It was made to match SelectionList type.
Changed